packages/openapi-to-graphql-cli/.prettierignore

```diff
@@ -1 +1,2 @@
-lib/
\ No newline at end of file
+lib/
+*.md
```

Comment:
Some .md files contain code snippits, which Prettier is smart enough to recognize and will add semicolons...